test/SlowTests/Issues/RavenDB_8369.cs

```diff
@@ -81,8 +83,12 @@ private async Task Tomstones_should_be_cleaned_properly_for_multiple_backup_task
                     session.Delete("foo/bar1");
                     session.SaveChanges();
                 }
-                var config1 = Backup.CreateBackupConfiguration(backupPath1, backupType: backupType, incrementalBackupFrequency: "* */6 * * *");
-                var config2 = Backup.CreateBackupConfiguration(backupPath2, backupType: backupType, incrementalBackupFrequency: "* */6 * * *");
+
+                int hour = (DateTime.Now.Hour + 1)%24;
```

Comment:
Why changing the cron expression fixes the test?

Reply:
Because the old cron expression backup2 sometimes runs automatically before those lines:
https://github.com/ravendb/ravendb/blob/638034afc5990ba49e13cc14a71fe0233c66491c/test/SlowTests/Issues/RavenDB_8369.cs#L102
https://github.com/ravendb/ravendb/blob/638034afc5990ba49e13cc14a71fe0233c66491c/test/SlowTests/Issues/RavenDB_8369.cs#L122
- It depense on when the time the test is running, because the meaning of the old corn expression is 
  “At every minute past every 6th hour” (6:00, 6:01, 6:02,.. , 12:00, 12:01, 12:02, .. ,18:00, 18:01, 18:02, 00:00, 00:01, 00:02,..).